app/models/book_adoption_eve_page.rb

```diff
@@ -6,6 +6,9 @@ class BookAdoptionEvePage < Fae::StaticPage
   def self.fae_fields
     {
       hero_image: { type: Fae::Image },
+      banner_tag: { type: Fae::TextField },
+      lede_text: { type: Fae::TextField },
```

Comment:
Do `banner_tag` and `lede_text` need to be removed from here now?

Reply:
I removed and added banner in the plan page and event page